cypress/e2e/Courses/Certificates/FreePlanAwardedCourseCertificate.feature

```diff
@@ -1,3 +1,4 @@
+@course @certificate @free-plan @business:low-impact
```

Comment:
Of course, downloading the certificate is important, but for the business is more important to focus on the unawarded ones (potential buyers). Furthermore, free users are not as important as paid users.